fireant/slicer/slicers.py

```diff
@@ -0,0 +1,80 @@
+import itertools
+
+from .queries import QueryBuilder
+
+
+def _match_items(left, right, key):
+    return all([a is not None
+                and b is not None
+                and getattr(a, key) == getattr(b, key)
+                for a, b in itertools.zip_longest(left, right)])
+
+class _Container(object):
+    def __init__(self, items):
```

Comment:
Please add docstrings for _Container and _match_items

Reply:
will do

Reply:
gonna remove `_match_items` and make it the `__eq__` function for `_Container`